IdentityCore/src/oauth2/MSIDWebviewFactory.m

```diff
@@ -84,13 +84,28 @@ - (MSIDWebviewSession *)systemWebviewSessionFromConfiguration:(MSIDWebviewConfig
     NSURL *startURL = [self startURLFromConfiguration:configuration requestState:state];
     NSURL *redirectURL = [NSURL URLWithString:configuration.redirectUri];
     
-    MSIDSystemWebviewController *systemWVC = [[MSIDSystemWebviewController alloc] initWithStartURL:startURL
-                                                                                 callbackURLScheme:redirectURL.scheme
-                                                                                  parentController:configuration.parentController
-                                                                                  presentationType:configuration.presentationType
-                                                                          useAuthenticationSession:useAuthenticationSession
-                                                                         allowSafariViewController:allowSafariViewController
-                                                                                           context:context];
+    MSIDSystemWebviewController *systemWVC;
+    if (@available(iOS 13.0, *))
+    {
+        systemWVC = [[MSIDSystemWebviewController alloc] initWithStartURL:startURL
```

Comment:
nit: do we need if else here? could we have a parameter that defaults to NO on older platforms?

Reply:
done